controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require("../models/topics.model");
+
+exports.getTopics = (req, res, next) => {
+  return fetchTopics()
+    .then((topics) => {
+      res.send({ topics });
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
a shorthand you can use for the catch is 
```js
.catch(next);
```
this will invoke next with the error